tests/plugins/magicline/manual/lineattribute.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: 1441, feature, 4.9.0, magicline
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, basicstyles, elementspath, magicline
+
+1. Hover the cursor over the editor and find a magic line pop-up.
```

Comment:
Calling it magicline pop-up may be a little misleading, I would go with something like:

> 1. Hover over different editor elements in a way that magic line is visible.

and basically this one step is enough. Then you will have **Expected** and **Actual**:

> Whole magic line element is red/blue.